test/pipe.js

```diff
@@ -50,23 +50,54 @@ assert.equal(shell.error(), null);
 assert.equal(result.toString(), shell.cat('resources/uniq/pipeSorted').toString());
 
 // Synchronous exec
-// TODO: add windows tests
-if (process.platform !== 'win32') {
+if (process.platform === 'win32') {
+  // Windows-specific
+
+  // Ensure the user does not have the wrong version of FIND. I.e.,
+  // require the following to return error.
+  result = shell.exec('CD..&FIND . -name no');
```

Comment:
I’m not sure how we would do that. The point is to detect people (like me) who like to put unix utilities such as [MSYS2](https://msys2.github.io/) in `PATH` and attempting to `npm test` instead of cleaning up their environment first. Basically, the test failure that you would get when you get `find(1)` instead of `FIND` would be quite confusing, so I think this check is worth it. `FIND` will error because the first argument is not wrapped in `""` whereas unix-like `find(1)` will exit successfully. Just finding the patch to `FIND` would return a path. I guess you could do something like trying to determine if the returned path is in `%WINDIR%\System32` or not, but that seems a lot more fragile than actually testing the behavior.